packages/bpk-scrim-utils/src/bpk-scrim.scss

```diff
@@ -47,9 +47,5 @@
 
   &--appear-active {
     opacity: $bpk-scrim-opacity;
-
-    @include bpk-breakpoint-mobile {
-      opacity: $bpk-scrim-mobile-opacity;
```

Comment:
This value was 1, which is why, on mobile, it was causing it to reach full opacity befoore returning to the final opacity value of 0.7